playground/backend/internal/environment/environment_service.go

```diff
@@ -37,22 +36,18 @@ const (
 	cacheTypeKey                  = "CACHE_TYPE"
 	cacheAddressKey               = "CACHE_ADDRESS"
 	beamPathKey                   = "BEAM_PATH"
-	beamRunnerKey                 = "BEAM_RUNNER"
-	SLF4jKey                      = "SLF4J"
 	cacheKeyExpirationTimeKey     = "KEY_EXPIRATION_TIME"
 	pipelineExecuteTimeoutKey     = "PIPELINE_EXPIRATION_TIMEOUT"
 	protocolTypeKey               = "PROTOCOL_TYPE"
 	defaultProtocol               = "HTTP"
 	defaultIp                     = "localhost"
 	defaultPort                   = 8080
 	defaultSdk                    = pb.Sdk_SDK_JAVA
-	defaultBeamSdkPath            = "/opt/apache/beam/jars/beam-sdks-java-harness.jar"
+	DefaultBeamJarsPath           = "/opt/apache/beam/jars/*"
```

Comment:
I suppose the name of this constant should be `defaultBeamJarsPath` starting with a lower case?

Reply:
far I understand this constant exported only for unit test if it is true - please think how to avoid it

Reply:
Done